rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidPatientRegisterFragment.java

```diff
@@ -49,9 +54,6 @@ protected String getDefaultSortQuery() {
     @Override
     public void onSyncComplete(FetchStatus fetchStatus) {
         super.onSyncComplete(fetchStatus);
-        RDTApplication.getInstance()
-                .getContext()
-                .allSharedPreferences()
-                .savePreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP, String.valueOf(System.currentTimeMillis()));
+        LocalBroadcastManager.getInstance(getActivity()).sendBroadcast(new Intent(PatientRegisterActivity.ACTION_UPDATE_LATEST_SYNC_DATE));
```

Comment:
This can be removed.

Reply:
if i remove this then we will lose the real-time updates

Reply:
refer to #899 (comment)